iwyu_output.cc

```diff
@@ -1978,18 +1978,20 @@ size_t PrintableDiffs(const string& filename,
 
   // First, check if there are no adds or deletes.  If so, we print a
   // shorter summary line.
-  bool no_adds_or_deletes = true;
-  for (const auto& key_line : sorted_lines) {
-    const OneIncludeOrForwardDeclareLine* line = key_line.second;
-    if ((line->is_desired() && !line->is_present()) || // add
-        (line->is_present() && !line->is_desired())) { // delete
-      no_adds_or_deletes = false;
-      break;
+  if (!GlobalFlags().force_comments) {
```

Comment:
I like the way this avoids extra work to determine if there are no adds or deletes, but it's annoying that the diff becomes so large due to reindentation. I think we should take the small perf hit and do:
```
if (no_adds_or_deletes && !GlobalFlags().force_comments) {
      output = "\n(" + filename + " has correct #includes/fwd-decls)\n";
      return 0;
}
```